src/server/controllers/changePassword.ts

```diff
@@ -109,6 +115,9 @@ const oktaIdxApiPasswordHandler = async ({
 				});
 			}
 
+			// track the password change metric
+			changePasswordMetric(path, 'Success', true);
```

Comment:
Should something be done with the return value of this function? Unless I'm missing something it seems like all this function does is return a string?

Reply:
these are fire and forget promises: https://github.com/guardian/gateway/blob/main/src/server/lib/trackMetric.ts#L24

so we don't care if they succeed/fail and they don't throw either

Reply:
Shouldn't it be something like this instead then?
 
```suggestion
			trackMetric(changePasswordMetric(path, 'Success', true));
```

Reply:
As `[changePasswordMetric](https://github.com/guardian/gateway/blob/74e80ef6a07113f143cc2da9a73fed54d1c9e1ff/src/server/models/Metrics.ts#L93-L105)` doesn't seem to actually call `trackMetric` itself.

Reply:
Ah yes!

Very good spot, it should be that, it's what we do elsewhere e.g. for `emailSendMetric`

Will update